### PR TITLE
Add new entry for AYA Innovation Labs as adopter

### PR DESCRIPTION
**Pull request by Yashsharma1911**
**Notes for Reviewers**

This PR fixes #

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Yashsharma1911** [comment]
Hey @heinhtetwin, I have updated "AYA Innovation Labs" as Meshery adopters. Could you please review it to ensure that I have added the correct information?

**Comment by heinhtetwin** [comment]
Hello @Yashsharma1911 Thank you for adding AYA Innovation Labs as an adopter in Meshery. Can you please change the website address to https://ayapay.com as we are the backbone tech house of AYAPAY. Thank you

**Review by leecalcote — DISMISSED** [review]
This is awesome, @Yashsharma1911, @heinhtetwin.

File: ADOPTERS.md [hunk 1]
```diff
@@ -17,6 +17,7 @@
 - [Red Hat](https://www.redhat.com)
 - [Metabit Trading](https://www.metabit-trading.com/)
 - [PITS Global Data Recovery Services](https://www.pitsdatarecovery.net/)
+- [AYA Innovation Labs](https://www.ayabank.com/)
```
Comment (hamza-mohd) [inline ADOPTERS.md:20]:
```suggestion
- [AYA Innovation Labs](https://ayapay.com/)
```

I can help. Here we are. Good to go.

**Review by leecalcote — APPROVED** [review]
Very 😎

**Comment by Yashsharma1911** [comment]
Thankyou everyone ^^

**Pull request merged**